### PR TITLE
Can start chain with gates closed

### PR DESCRIPTION
**Pull request by rhuttman**

### CODE REVIEW

File: pallets/valve/src/mock.rs [hunk 1]
```diff
@@ -199,13 +200,21 @@ impl ExtBuilder {
...
 	pub(crate) fn build(self) -> sp_io::TestExternalities {
 		let mut t = frame_system::GenesisConfig::default()
 			.build_storage::<Test>()
 			.expect("Frame system builds valid default genesis config");
 
 		GenesisBuild::<Test>::assimilate_storage(
-			&pallet_valve::GenesisConfig { start_with_valve_closed: self.valve_closed },
+			&pallet_valve::GenesisConfig {
+				start_with_valve_closed: self.valve_closed,
+				closed_gates: self.closed_gateds,
```
Comment (justinzhou93) [inline pallets/valve/src/mock.rs:216]: nit spelling

File: pallets/valve/src/mock.rs [hunk 2]
```diff
@@ -199,13 +200,21 @@ impl ExtBuilder {
 		self
 	}
 
+	pub(crate) fn with_gate_closed(mut self, g: Vec<u8>) -> Self {
+		self.closed_gateds = vec![g];
```
Comment (justinzhou93) [inline pallets/valve/src/mock.rs:204]: nit spelling gateds

File: node/src/chain_spec/neumann.rs [hunk 3]
```diff
@@ -72,6 +72,7 @@ pub fn development_config() -> ChainSpec {
 				get_account_id_from_seed::<sr25519::Public>("Alice"),
 				endowed_accounts,
 				DEFAULT_PARA_ID.into(),
+				vec![],
```
Comment (justinzhou93) [inline node/src/chain_spec/neumann.rs:75]: lol this is all it took to originate an account?

**Review by justinzhou93 — APPROVED** [review]
nits, otherwise lgtm

**Review by irsal — APPROVED** [review]

**Pull request merged**